packages/backend/src/database/migrations/20220330140655_alter_organisation_add_chart_colours.ts

```diff
@@ -0,0 +1,11 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.raw(`ALTER TABLE organizations
+        ADD COLUMN chart_colors _text ;`);
+}
+
+export async function down(knex: Knex): Promise<void> {
+    await knex.raw(`ALTER TABLE organizations 
+    DROP COLUMN chart_colors CASCADE;`);
```

Comment:
What does `CASCADE` do on a column drop?

Reply:
Not our case, but 
```
If the column that you want to remove is used in other database objects such as [views](https://www.postgresqltutorial.com/postgresql-views/), [triggers](https://www.postgresqltutorial.com/postgresql-triggers/), [stored procedures](https://www.postgresqltutorial.com/postgresql-stored-procedures/), etc., you cannot drop the column because other objects are depending on it. In this case, you need to add the CASCADE option to the DROP COLUMN clause to drop the column and all of its dependent objects:
```

Reply:
Never knew that!